perf/stability/rabbitmq/templates/rabbitmq.yaml

```diff
@@ -0,0 +1,301 @@
+---
+# Source: rabbitmq/templates/secrets.yaml
```

Comment:
Can you put a URL link here?
How do we keep these up to date?

Reply:
Thanks, thought I did